ECommerceAPI/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
```

Comment:
why not set up a pylint that could help you to be consistent in your code e.g. single and double quote

Reply:
will do